plugins/woocommerce/tests/api-core-tests/tests/settings/settings-crud.test.js

```diff
@@ -1629,34 +1676,44 @@ test.describe( 'Settings API tests: CRUD', () => {
 			const responseJSON = await response.json();
 			expect( response.status() ).toEqual( 200 );
 			expect( Array.isArray( responseJSON ) ).toBe( true );
-			expect( responseJSON ).toEqual(
-				expect.arrayContaining( [
-					expect.objectContaining( {
-						id: 'woocommerce_cart_page_id',
-						label: 'Cart page',
-						description: 'Page contents: [woocommerce_cart]',
-						type: 'select',
-						default: '',
-						tip: 'Page contents: [woocommerce_cart]',
-						value: expect.any( String ),
-						options: expect.any( Object ),
-					} ),
-				] )
-			);
-			expect( responseJSON ).toEqual(
-				expect.arrayContaining( [
-					expect.objectContaining( {
-						id: 'woocommerce_checkout_page_id',
-						label: 'Checkout page',
-						description: 'Page contents: [woocommerce_checkout]',
-						type: 'select',
-						default: expect.any( Number ),
-						tip: 'Page contents: [woocommerce_checkout]',
-						value: expect.any( String ),
-						options: expect.any( Object ),
-					} ),
-				] )
-			);
+
```

Comment:
These two aren't present on the external host.  There's a Cart page and a Checkout page, but they aren't being returned in the JSON response.